### PR TITLE
Add 'redistribute' support to router_ospf_vrf

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
* most of this code was repurposed from bgp_af

* 'redistribute maximum-prefix' is not currently supported

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Changelog
 ### New Cisco Resources
 
 ### Added
+* Extend router_ospf_vrf with attributes:
```
Comment (mikewiebe) [inline CHANGELOG.md]: *nit `/attributes/attribute/`

**Review by mikewiebe — COMMENTED** [review]
Overall looks good.  Just one small *nit* and the Travis issues.

File: CHANGELOG.md (hunk 1 above)
Comment (chrisvanheuveln) [reply CHANGELOG.md]: ✔️

**Review by mikewiebe — APPROVED** [review]
👍

**Pull request merged**